usaspending_api/etl/management/commands/elasticsearch_indexer.py

```diff
@@ -72,13 +72,11 @@ def add_arguments(self, parser):
             metavar="[1-70]",
         )
         parser.add_argument(
-            "--partitions",
+            "--partition-size",
             type=int,
-            help="Set how much the data are partitioned. "
-            "NOTICE: large datasets may need indexes or other performance "
-            " optimizations if this is changed",
-            default=10000,
-            metavar="(default: 10,000)",
+            help="Set the upper-limit of a single data partition. ",
```

Comment:
The help docs throws me a bit, thinking only a limit, and not a target. 

```suggestion
            help="Set the desired records in a single data partition. Skew on primary key values may yield less per partition, but no partitions will be more than this size.",
```

Reply:
You didn't like this suggestion? The text you put in though isn't accurate, since with modulo, the partitions will never be bigger than the given size.

```
            help="Set the target size of a single data partition. A partition "
            "might be slightly larger or slightly smaller depending on the "
            " distribution of the data to process"
```

Reply:
Talked -- the partition calculation may be subject to change and may e.g. try to create "balanced" batches. e.g. if 101 records and batch size is 10 ... make batch size to 11 instead to get a better fill of each partition.